themes/cms-forms/templates/SilverStripe/Forms/CMSTabSet.ss

```diff
@@ -5,15 +5,17 @@
 <%-- e.g. through LeftAndMain_EditForm.ss. --%>
 
 <div $AttributesHTML>
-	<% loop $Tabs %>
-		<% if $Tabs %>
-			$FieldHolder
-		<% else %>
-			<div $AttributesHTML>
-				<% loop $Fields %>
-					$FieldHolder
-				<% end_loop %>
-			</div>
-		<% end_if %>
-	<% end_loop %>
+	<div class="tab-content">
+        <% loop $Tabs %>
+            <% if $Tabs %>
+                $FieldHolder
+            <% else %>
+                <div $getAttributesHTML("class") class="tab-pane $extraClass">
```

Comment:
`.tab-pane` has no styling implications here and isn't used for any js other than the validation tab-icon logic.